// 10_exception_handing_6.cpp